backend/Auth/ResetPassword.ts

```diff
@@ -64,3 +68,50 @@ router.post(
     return res.status(204).end();
   }
 );
+
+router.get(
+  "/validate-token/:token",
+  param("token").exists(),
```

Comment:
I think that express itself will guarantee that the `token` param exists, since otherwise express wouldn't match the incoming request to this route. Could could do `param("token").isString().notEmpty()`, but I think that also would be redundant because express params always start as strings and if it were empty it again would not have matched the incoming request to this route.

So I think this is unnecessary, but it also isn't hurting. It's a good practice in general to add validation for all params, query params, and body from the request, so I think leaving this here is fine.